File: app/src/main/java/com/thathurleyguy/lightning/models/Command.java
package com.thathurleyguy.lightning.models;

public class Command {
    public static final String POWER = "power";
    public static final String VOLUME_UP = "volume_up";
    public static final String VOLUME_DOWN = "volume_down";
    public static final String INPUT_GAME = "game";
    public static final String INPUT_XBMC = "video1";
    public static final String XBMC_UP = "up";
    public static final String XBMC_DOWN = "down";
    public static final String XBMC_LEFT = "left";
    public static final String XBMC_RIGHT = "right";
    public static final String XBMC_SELECT = "select";

    public static final String XBMC_PAUSE = "pause";
    public static final String XBMC_BACK = "back";
    public static final String XBMC_MENU = "osd";

    private String command;

    public Command() {

    }

    public Command(String command) {
        this.command = command;
    }
}

File: app/src/main/java/com/thathurleyguy/lightning/models/Command.kt
package com.thathurleyguy.lightning.models

public class Command(private val command: String) {

    companion object {
        public val POWER: String = "power"
        public val VOLUME_UP: String = "volume_up"
        public val VOLUME_DOWN: String = "volume_down"
        public val INPUT_GAME: String = "game"
        public val INPUT_XBMC: String = "video1"
        public val XBMC_UP: String = "up"
        public val XBMC_DOWN: String = "down"
        public val XBMC_LEFT: String = "left"
        public val XBMC_RIGHT: String = "right"
        public val XBMC_SELECT: String = "select"

        public val XBMC_PAUSE: String = "pause"
        public val XBMC_BACK: String = "back"
        public val XBMC_MENU: String = "osd"
    }
}
